client/src/components/PitchStatus/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC, ReactElement } from 'react';
+
+import { FieldTag } from '..';
+
+interface PitchStatusProps {
+  date: Date;
+}
+
+const PitchStatus: FC<PitchStatusProps> = ({ date }): ReactElement => {
+  if (new Date(date) < new Date()) {
+    return (
+      <>
+        <FieldTag name="Published" hexcode="#E9F4E7" />
```

Comment:
Not the way to do this. Please use content and then add the stuff back to the FieldTag and then in the global styles, make a class for published-tag-color and in-progress-tag-color and assign these hexcodes.